src/openzaak/components/besluiten/openapi.yaml

```diff
@@ -1368,12 +1368,10 @@ paths:
           format: uri
       - name: informatieobject
         in: query
-        description: URL-referentie naar het INFORMATIEOBJECT (in de Documenten API)
-          waarin (een deel van) het besluit beschreven is.
+        description: ''
         required: false
         schema:
           type: string
-          format: uri
```

Comment:
And this.

See: https://github.com/VNG-Realisatie/gemma-zaken/blob/master/api-specificatie/brc/1.0.x/openapi.yaml#L1400